index.js

```diff
@@ -1,17 +1,28 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+
 module.exports = {
   name: 'ember-cli-hoverintent',
 
+  treeForVendor: function () {
```

Comment:
You need to use and merge the passed in tree also, otherwise your vendor shim will be missing